src/extension/extension.ts

```diff
@@ -490,6 +490,14 @@ export class RunmeExtension {
 
     if (kernel.isFeatureOn(FeatureName.RequireStatefulAuth)) {
       await StatefulAuthProvider.instance.ensureSession()
+      StatefulAuthProvider.instance.currentSession().then(async (session) => {
```

Comment:
@pastuxso don't we need to `await` this here?

Reply:
Let me know if you have further suggestions!